cmd/destroy.go

```diff
@@ -15,8 +15,13 @@ var (
 		Flags: []cli.Flag{&appFlag,
 			&cli.BoolFlag{Name: "force", Usage: "Force destroy without asking for a confirmation /!\\"},
 		},
-		Usage:       "Destroy an app /!\\",
-		Description: "Destroy an app /!\\ It is not reversible\n	Example:\n    'scalingo destroy my-app'\n    'scalingo -a my-app destroy --force'\n	",
+		Usage: "Destroy an app /!\\",
+		Description: `Destroy an app /!\\ It is not reversible	
+
+Example:
+ $ scalingo destroy my-app
+ $ scalingo -a my-app destroy --force'
```

Comment:
suggestion: removed `:` to be consistent with the `create` command help description. But we may want to update if it's not consistent with the other commands.
suggestion: to ease the copy-paste of the full line, I would remove `$`
convention: we usually use long options for documentations.

```suggestion
Examples
 scalingo destroy my-app
 scalingo --app my-app destroy --force
```